src/main/resources/db/changelog/v.0.2/db.changelog-user.yaml

```diff
@@ -48,5 +48,18 @@ databaseChangeLog:
                   constraints:
                     nullable: false
 
-
-
+  - changeSet:
+      id: 12042021-add-colomn-for-soft-delete-to-user
+      author: nikita_bogurov
+      changes:
+        - addColumn:
+            tableName: user
+            columns:
+              name: u_deleted
+              type: BOOLEAN
+        - addDefaultValue:
```

Comment:
It's possible to add default value in the addColumn tag

Reply:
Done